src/SixLabors.Fonts/TextMeasurer.cs

```diff
@@ -22,7 +21,7 @@ public static class TextMeasurer
         /// <param name="text">The text.</param>
         /// <param name="options">The style.</param>
         /// <returns>The size of the text if it was to be rendered.</returns>
-        public static SizeF Measure(string text, RendererOptions options)
+        public static FontRectangle Measure(string text, RendererOptions options)
```

Comment:
What is the value of `rect.X` and `rect.Y` here? Are we sure this is better, than returning `Vector2` here?

Reply:
Its actually the top left position based on the render... no problem with just setting X,Y to 0,0 either, couldn't decide when I made the change so I left it as the render position.

The benefit to using this instead of `Vectort2` is for anyone using `var` instead of the type directly then the returned type still has a `Width` and `Height` property which is a much more intuitive experience and thus source code compatible.

Reply:
Man... I can't make up my mind about `FontRectangle`...